src/tools/spark-sstfile-generator/src/main/scala/com/vesoft/tools/SstFileOutputFormat.scala

```diff
@@ -0,0 +1,233 @@
+package com.vesoft.tools
+
+import java.io.{File, IOException}
+
+import com.vesoft.tools.SparkSstFileGenerator.GraphPartitionIdAndKeyValueEncoded
+import com.vesoft.tools.VertexOrEdgeEnum.VertexOrEdgeEnum
+import javax.xml.bind.DatatypeConverter
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{FileSystem, Path}
+import org.apache.hadoop.mapred.InvalidJobConfException
+import org.apache.hadoop.mapreduce._
+import org.apache.hadoop.mapreduce.lib.output.FileOutputFormat
+import org.apache.hadoop.mapreduce.security.TokenCache
+import org.rocksdb.{EnvOptions, Options, SstFileWriter}
+import org.slf4j.LoggerFactory
+
+import scala.collection.mutable
+import scala.sys.process._
+
+/**
+  * Custom outputFormat, which generate a sub dir per partition per worker:
+  *
+  * worker_node1
+  * |
+  * |-sstFileOutput
+  *   |
+  *   |--1
+  *   |  |
+  *   |  |——vertex-${FIRST_KEY1}.data
```

Comment:
Why not "sst" as suffix.

And @darionyaphet,  please take it as your reference when you do download feature.

Reply:
Good point.